ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/General.hs

```diff
@@ -200,6 +201,9 @@ runTestNetwork ::
      , Show (LedgerView (BlockProtocol blk))
      )
   => TestConfig
+  -> EpochSize
+  -- ^ Temporary: until we start testing the hard fork combinator, we just
+  -- support a single 'EpochSize'. See also comments for 'tnaEpochInfo'.
```

Comment:
This should be part of `TestConfig` or `TestConfigBlock`.

Reply:
`TestConfigBlock`  is block specific. `TestConfig`  would make sense in _principle_, but it gets generated independent from `k`, and so I can't produce a value for `EpochSize` there.

Reply:
Solution: `TestConfig` should include `SecurityParam`. But okay, let's not do that in this PR.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/1735

Reply:
But see https://github.com/input-output-hk/ouroboros-network/issues/1735#issuecomment-593822774.